deps.edn

```diff
@@ -1,6 +1,6 @@
 {:paths ["src" "resources"]
  :deps  {org.clojure/clojure        {:mvn/version "1.11.1"}
-         techascent/tech.ml.dataset {:mvn/version "7.007"}
+         techascent/tech.ml.dataset {:mvn/version "7.010"}
```

Comment:
there is a 7.011 now. :-)

Reply:
Have moved TMD & TC to a `:dev` alias.
TMD v7.011 not on maven yet (so hadn't spotted) but is on clojars so bumped.